pages/feedback.tsx

```diff
@@ -4,19 +4,23 @@ import ReviewBubble from "../components/ReviewBubble";
 import axios from "axios"
 import HamburgerMenu from "../components/HamburgerMenu";
 import LoadingIcons from "react-loading-icons";
+import { Session } from "next-auth";
+import AccountPanel from "../components/AccountPanel";
+import { GetServerSidePropsContext } from "next";
+import { getSession } from "next-auth/react";
 
-export default function Feedback() {
-    const [search, setSearch] = useState("");
-    // const filteredReviewsArray = reviewsArray.filter(r=> r.text.toLowerCase().includes(search.toLowerCase()) ||
-    // r.subject.toLowerCase().includes(search.toLowerCase()) ||
-    // r.date.toLowerCase().includes(search.toLowerCase()) ||
-    // r.level.toLowerCase().includes(search.toLowerCase())  )
+export async function getServerSideProps(context: GetServerSidePropsContext) {
+    const session = await getSession(context)
+    if (!session) return { props: {} }         //go back to sign in page
```

Comment:
I'm confused about this comment - it doesn't look like you go back to the sign in page, you just don't show the "logged in" information? I think unauthorised users should be allowed to view the reviews right?

Reply:
Yeah the comment is wrong, it just stays on the current page when you log out